LESSON2.md

```diff
@@ -361,8 +361,3 @@ stdin.on('end', () => {
 
 *This is exactly the same as our read stream example at the top of this readme, as
 process.openStdin() is a read stream!*
-
-### Project
-
-With what we have covered in these two lessons, we now have everything we need to build some command-line tools! Move onto
```

Comment:
@joeylouise I'm happy with all the other changes (thanks! ✨ ) but am unsure if we need to remove the project.md. I'd prefer to keep it in for the benefit of those who use this workshop outside of a FAC classroom. 

Could you tell the students to finish where they are if they get to the end of lesson 2 as the project will be covered in the research afternoon? If you wanted to include the project in the master-reference you could provide a link to the project file here from the research-afternoon.md file.